src/main/java/com/hubspot/jackson/jaxrs/PropertyFiltering.java

```diff
@@ -9,4 +9,5 @@
 @Retention(RetentionPolicy.RUNTIME)
 public @interface PropertyFiltering {
   String using() default "property";
+  String[] defaultProperties() default {};
```

Comment:
I wasn't expecting them to always be added, I was thinking `defaultProperties()` meant they would only be used if nothing was specified in the querystring. Maybe `always()` is a less ambiguous name?